source/Generic/ExtraMetadataToolsVideo/ExtraMetadataToolsVideo.psm1

```diff
@@ -59,15 +59,241 @@ function GetGameMenuItems
     $menuItem11.FunctionName = "Update-AssetsStatusGameDatabase"
     $menuItem11.MenuSection = "Extra Metadata tools"
 
-    return $menuItem, $menuItem2, $menuItem3, $menuItem9, $menuItem10, $menuItem4, $menuItem5, $menuItem6, $menuItem7, $menuItem8, $menuItem11
+    $menuItem12 = New-Object Playnite.SDK.Plugins.ScriptGameMenuItem
+    $menuItem12.Description = [Playnite.SDK.ResourceProvider]::GetString("LOCExtra_Metadata_tools_MenuItemInvoke-ViewSettingsDescription")
+    $menuItem12.FunctionName = "Invoke-ViewSettings"
+    $menuItem12.MenuSection = "Extra Metadata tools"
+
+    return $menuItem, $menuItem2, $menuItem3, $menuItem9, $menuItem10, $menuItem4, $menuItem5, $menuItem6, $menuItem7, $menuItem8, $menuItem11, $menuItem12
+}
+
+function Invoke-ViewSettings
+{
+    param(
+        $scriptGameMenuItemActionArgs
+    )
+    
+    $settings = Get-Settings
+
+    # Load assemblies
+    Add-Type -AssemblyName PresentationCore
+    Add-Type -AssemblyName PresentationFramework
+
+    # Set Xaml
+    [xml]$Xaml = @"
+<Grid xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+    <Grid.Resources>
+        <Style TargetType="TextBlock" BasedOn="{StaticResource BaseTextBlockStyle}" />
+    </Grid.Resources>
+
+    <DockPanel Margin="20">
+        <DockPanel DockPanel.Dock="Bottom" LastChildFill="False">
+            <Button Name="ButtonCancel" Margin="10,0,0,0" DockPanel.Dock="Right" VerticalAlignment="Bottom"/>
+            <Button Name="ButtonSave" Margin="0,0,0,0" DockPanel.Dock="Right" VerticalAlignment="Bottom"/>
+        </DockPanel>
+        <StackPanel Grid.Row="0" DockPanel.Dock="Top">
+
+            <TextBlock Text="{DynamicResource LOCExtra_Metadata_tools_FFMPEGP_PATH}" Name="TBLffmpegPath" Margin="0,0,0,3"/>
+            <DockPanel Margin="0,0,0,10">
+                <Button Name="BTNffmpegPath" Margin="10,0,0,0" DockPanel.Dock="Right">...</Button>
```

Comment:
In all buttons, please just set the Content just by naming the property like `Content="..."` instead of opening
In this case, it would be
`<Button Name="BTNffmpegPath" Margin="10,0,0,0" DockPanel.Dock="Right" Content="..." />`

Reply:
Done in 04a0d39d82237c61cbdc29853f1fe31fe68bbe30